### ISSUE TITLE
Support for empty attribute_map.yml

### ISSUE THREAD

**Issue by SirRawlins** [apokalipto/devise_saml_authenticatable#158]
I currently use this gem for very simple SSO, which uses the Subject to identify the user, and doesn't need to map any other values.

Removing the `attribute_map.yml` file raises an emption.

> No such file or directory @ rb_sysopen - /example/config/attribute-map.yml

Providing an empty file also raises an exception.

> undefined method `has_key?' for false:FalseClass

it appears to be parsed as '`false` rather than an empty hash.

```ruby
def attribute_map_for_environment
  attribute_map = YAML.load(File.read("#{Rails.root}/config/attribute-map.yml"))
  if attribute_map.has_key?(Rails.env)
    attribute_map[Rails.env]
  else
    attribute_map
  end
end
```

Is there any objection to a PR which allows for this? So that non-existent or empty files are parsed as an empty hash?

**Comment by adamstegman** [comment]
I've definitely run into this bug multiple times, it's really annoying that `YAML` returns false in this circumstance. A PR would be really helpful, thank you!

**Comment by luke-zhou** [comment]
This is no longer an issue. Looks like it has been fixed

**Comment by adamstegman** [comment]
Good point, this was fixed in #162. Thanks!
